frontend_app_gallery/test/component/Carousel.spec.js

```diff
@@ -36,11 +36,11 @@ describe('<Carousel />', () => {
       })
 
       it('should display the right message when it is a folder', () => {
-        expect(wrapper.find('.gallery__noContent').text()).to.contain('There isn\'t any previewable content at that folder\'s root.')
+        expect(wrapper.find('.gallery__noContent').text()).to.contain(`There isn't any previewable content at that folder's root.`)
```

Comment:
theses are back-ticks 
It is for string interpolation and multi-lines strings. It should not be used for simple strings because might require more processing.
single quote: '
double quote: "
back tick: `